package.json

```diff
@@ -31,7 +31,7 @@
     "@types/graphql": "^14.0.0",
     "@types/mocha": "^2.2.39",
     "@types/node": "^8.0.28",
-    "@types/sinon": "^5.0.1",
+    "@types/sinon": "5.0.2",
```

Comment:
Sorry, I didn't realize you had fixed this issue in your branch, please feel free to pin this when fixing the conflicts.